### DISCUSSION TITLE
:partying_face: Jupytext now supports Xonsh!

### DISCUSSION THREAD

**Discussion by jsquaredosquared** [xonsh/xonsh#5401] [Show and tell]
Just thought I'd share in case anyone ever finds it useful :smile: 

https://jupytext.readthedocs.io/en/latest/languages.html

**Comment by anki-code** [comment]
Hey @jsquaredosquared ! That's awesome! Could you please provide more examples and use cases?

I will add note about this into xonsh news file.

**Reply by jsquaredosquared** [reply]
A very simple use case is to pair a xonsh .ipynb file with a xonsh .xsh file. I then prototype my script by writing notebook cells. When I am done, the notebook is already synced with and converted to a xonsh script which I can run. I just find it a bit easier to work on the script by building it up piece by piece from notebook cells.

I'm sure Jupytext can do more, this is just one way I found useful :D

**Reply by anki-code** [reply]
> I just find it a bit easier to work on the script by building it up piece by piece from notebook cells.

I agree. PR with news - https://github.com/xonsh/xonsh/pull/5402
